src/components/VaccinationEligibilityBlock/ButtonBlock.tsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import OpenInNewIcon from '@material-ui/icons/OpenInNew';
+import { EventAction, EventCategory } from 'components/Analytics';
+import { EmailAlertIcon } from 'components/EmailAlertsFooter/EmailAlertsFooter.style';
+import { StyledLinkButton, Wrapper } from './ButtonBlock.style';
+
+const ButtonBlock: React.FC<{ signupLink?: string | null }> = ({
+  signupLink,
+}) => {
+  const sharedTrackingProps = {
+    trackingCategory: EventCategory.VACCINATION,
+    trackingAction: EventAction.CLICK_LINK,
+  };
+
+  return (
+    <Wrapper>
+      <StyledLinkButton
+        to="#shareContainer"
+        {...sharedTrackingProps}
+        trackingLabel="Vaccination alerts"
+        startIcon={<EmailAlertIcon />}
+        aria-label="Vaccination alerts signup"
+      >
```

Comment:
Actually I forgot to implement the HashLink version for the `LinkButton` component, this might not work. 😢  - I will test on the preview link